chat_logger/admin.py

```diff
@@ -39,5 +40,10 @@ class Log5XXErrorAdmin(admin.ModelAdmin):
     )
     # radio_fields = {'status': admin.VERTICAL}
 
+    def mark_resolved(self, request, queryset):
```

Comment:
We can also add such action out of the class and use it to make it available to multiple model admins.